mrt/utils.py

```diff
@@ -180,31 +179,24 @@ def read_file(f):
     f.close()
 
 
-def generate_excel(header, rows, filename):
-    style = xlwt.XFStyle()
-    normalfont = xlwt.Font()
-    headerfont = xlwt.Font()
-    headerfont.bold = True
-    style.font = headerfont
-
-    wb = xlwt.Workbook(encoding='utf-8')
-    ws = wb.add_sheet('Sheet 1')
-    row = 0
-
-    for col in range(len(header)):
-        ws.row(row).set_cell_text(col, header[col], style)
+def generate_export_excel(header, rows, filename):
+    workbook = Workbook()
+    sheet = workbook.active
 
-    style.font = normalfont
+    for col_idx, name in enumerate(header, 1):
+        cell_col_letter = openpyxl.utils.get_column_letter(col_idx)
+        sheet.cell(row=1, column=col_idx).value = name
+        sheet.column_dimensions[cell_col_letter].width = len(name) + 1
```

Comment:
We should be able to set the a bold font for headers here with openpyxl as we did with xlwt